packages/valdueza-storybook/package.json

```diff
@@ -17,7 +17,8 @@
     "extends": "react-app"
   },
   "dependencies": {
-    "medulas-react-components": "^0.2.0"
+    "medulas-react-components": "^0.2.0",
+    "babel-loader": "^8.0.6"
```

Comment:
Good! Context: babel-loader was removed as an explicit dependency from the root package.json in https://github.com/iov-one/ponferrada/pull/484/commits because the latest version of react-scripts complained and rightfully noted, that we should not install babel-loader on our own. Instead it is pulled in as a depdendecy from `react-scripts` and `@iov/browser-extension-react-scripts`. But here we need it as a runtime dependency for webpack to find it.